### DISCUSSION TITLE
Changer le libellé dans le cas du : "reuse_card_proposal"

### DISCUSSION THREAD

**Discussion by Jzano** [PayGreen/documentation#22] [Q&A] [answered]
Bonjour,
Est-il possible de changer le texte pour la validation de réutilisation de la carte bancaire ?

Je m'explique dans notre implémentation, cette notion est utilisée afin de pouvoir capturer un paiement si et seulement si le client n'honore pas sa réservation.
Or, le texte affiché porte à confusion et créer le doute chez le client final.

Nous souhaiterions pouvoir modifier ce texte pour être plus précis sur les futurs usages de la carte bancaire en accord avec les conditions générales de vente du vendeur.
Merci

**Comment by gaelDolle** [comment, marked as answer]
Bonjour,

Dans le cadre de l'utilisation de PaygreenJS, vous pouvez demander un audit auprès de notre service client afin de gérer vous même la case à cocher. Vous pourrez, après modification de votre configuration de boutique, utiliser la fonction paygreenjs.updateConsent(boolean) afin de gérer la checkbox.

Je vous laisse prendre contact : [Service client](https://paygreen.atlassian.net/servicedesk/customer/portal/9)

Cordialement.
